engine.jackson/src/test/java/org/opencds/cqf/cql/engine/execution/CqlListOperatorsTest.java

```diff
@@ -12,12 +14,8 @@
 import java.util.Collections;
 import java.util.List;
 
-import org.opencds.cqf.cql.engine.elm.execution.EquivalentEvaluator;
-import org.opencds.cqf.cql.engine.exception.InvalidOperatorArgument;
-import org.opencds.cqf.cql.engine.runtime.DateTime;
-import org.opencds.cqf.cql.engine.runtime.Time;
-import org.testng.Assert;
-import org.testng.annotations.Test;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
```

Comment:
Related to the above, we've generally tried to follow the Google style guide:
https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing

Reply:
Fixed all imports. :)